ssh/tailssh/tailssh.go

```diff
@@ -1376,15 +1385,18 @@ func (ss *sshSession) startNewRecording() (_ *recording, err error) {
 			// it. Then we can (1) make the cmd, (2) start the
 			// recording, (3) start the process.
 		},
+		SSHUser:   ss.conn.info.sshUser,
+		LocalUser: ss.conn.localUser.Username,
+		SrcNode:   ss.conn.info.node.Name,
+		SrcNodeID: ss.conn.info.node.StableID,
 	})
 	if err != nil {
-		f.Close()
+		pw.Close()
```

Comment:
add a `defer pw.Close()` above instead?

Reply:
no, this function just creates the recorder. We can't close it here.